torch_geometric/loader/node_loader.py

```diff
@@ -89,9 +92,17 @@ def __init__(
         self.input_data = InputData(input_nodes, input_time)
         self.transform = transform
         self.filter_per_worker = filter_per_worker
+        # CPU Affinitization for loader and compute cores
```

Comment:
Can we add a TODO here to unify this behind a `BaseDataLoader` class?

Reply:
added